src/components/CoinInfo.js

```diff
@@ -0,0 +1,122 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const CoinInfo = ({
+  image,
+  price,
+  name,
+  symbol,
+  marketCap,
+  marketCapRank,
+  totalVolume,
+  high24h,
+  low24h,
+  priceChange24h,
+  marketCapChange24h,
+  totalSupply,
+}) => (
+  <>
+    <div className="coinDetails">
```

Comment:
- [ ] Great work doing the adjustments. However, there are some aspects you can improve to make the design more appealing.
  - Make the `coinInfo` container cover the whole section.
  -  Increase the padding of the sections I have highlighted below  and make their width to cover `100%`
  
![image](https://user-images.githubusercontent.com/98453979/227651358-849c697d-1702-4284-88bd-fbc3c500b1e7.png)

### Guideline

![image](https://user-images.githubusercontent.com/98453979/227651493-ad75bf7b-f0e7-4519-97eb-29b3de20bb60.png)

Reply:
@frank1738 Hello Franky. I make width 100% but always some space stay. I couldn't fix it, how should I do?